### ISSUE TITLE
Feature: Preservation of posts

### ISSUE THREAD

**Issue by yamikuronue** [SockDrawer/SockRPG#52]
Given I am entering a post
When I am logged out due to inactivity
Then I should be offered the chance to log back in
And my post should still be saved in its entirety

This is the most annoying thing about traditional forums for RP: your session expires for security reasons and you lose a 2,000 word post you've been working on the whole time.

**Comment by RaceProUK** [comment]
Browser Local Storage to the rescue? I can't think of another way of doing it except maybe the Discourse way, but obviously that won't work if sessions expire.

**Comment by yamikuronue** [comment]
Yeah, that was my thought :) This came out of a conversation with @Kaelas17 last night
